Check/BestPractices/Multisite.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * @file
+ * Contains \SiteAudit\Check\BestPractices\Multisite.
+ */
+
+class SiteAuditCheckBestPracticesMultisite extends SiteAuditCheckAbstract {
+  /**
+   * Implements \SiteAudit\Check\Abstract\getLabel().
+   */
+  public function getLabel() {
+    return dt('Multi-site');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getDescription().
+   */
+  public function getDescription() {
+    return dt('Detect multi-site configurations.');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultFail().
+   */
+  public function getResultFail() {
+    return dt('The following multi-site configuration(s) were detected: @list', array(
+      '@list' => implode(', ', $this->registry['multisites']),
+    ));
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultInfo().
+   */
+  public function getResultInfo() {
+    return $this->getResultFail();
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultPass().
+   */
+  public function getResultPass() {
+    return dt('No multi-sites detected.');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultWarn().
+   */
+  public function getResultWarn() {
+    if ($this->registry['multisite_enabled']) {
+      return dt('sites/sites.php is present but no multisite directories are present.');
+    }
+    else {
+      return dt('Multisite directories are present but sites/sites.php is not present.');
+    }
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getAction().
+   */
+  public function getAction() {
+    if ($this->score == SiteAuditCheckAbstract::AUDIT_CHECK_SCORE_FAIL) {
+      return dt('See https://pantheon.io/blog/drupal-multisite-much-ado-about-drupal-multisite for details.');
+    }
+    if ($this->score == SiteAuditCheckAbstract::AUDIT_CHECK_SCORE_WARN) {
+      if ($this->registry['multisite_enabled']) {
+        return dt('See https://www.drupal.org/node/2297419 for details on how to use multisite feature in Drupal 8.');
+      }
+      else {
+        return dt('Create sites.php file inside sites directory by copying example.sites.php file. See https://www.drupal.org/node/2297419 for more details.');
```

Comment:
`Inside the sites/ directory, copy example.sites.php to sites.php to create the configuration. See https://www.drupal.org/node/2297419 for details.`